### PR TITLE
Add new repo for kickstart data ingestion into dataworks aws

### PR DESCRIPTION
**Pull request by ashisprasad06**
Hello,

This PR is to create new repo dataworks_aws_ksr_adg to hold codes required for ingesting and generating analytical dataset for kickstart into Dataworks. This repo will be initially maintained by team goldeneye

### CODE REVIEW

File: dataworks-aws-kickstart-adg.tf [hunk 1]
```diff
@@ -0,0 +1,81 @@
+locals {
+  dataworks_aws_kickstart_adg_pipeline_name = "dataworks_aws_kickstart_adg"
+}
+
+resource "github_repository" "dataworks_aws_kickstart_adg" {
+  name             = "dataworks_aws_kickstart_adg"
```
Comment (dan-hill2802) [inline dataworks-aws-kickstart-adg.tf:6]:
```suggestion
  name             = "dataworks-aws-kickstart-adg"
```
we use underscores in Terraform resource names, but hyphens in repo names

Comment (ashisprasad06) [reply dataworks-aws-kickstart-adg.tf:6]: Okay let me make the changes

File: dataworks-aws-kickstart-adg.tf [hunk 2]
```diff
@@ -0,0 +1,81 @@
+locals {
+  dataworks_aws_kickstart_adg_pipeline_name = "dataworks_aws_kickstart_adg"
```
Comment (dan-hill2802) [inline dataworks-aws-kickstart-adg.tf:2]:
```suggestion
  dataworks_aws_kickstart_adg_pipeline_name = "kickstart-adg"
```

Comment (ashisprasad06) [reply dataworks-aws-kickstart-adg.tf:2]: Ok, thanks I have made the changes

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**